packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -1926,11 +1927,7 @@ function recursivelyTraverseMutationEffects(
   if (deletions !== null) {
     for (let i = 0; i < deletions.length; i++) {
       const childToDelete = deletions[i];
-      try {
-        commitDeletionEffects(root, parentFiber, childToDelete);
-      } catch (error) {
-        captureCommitPhaseError(childToDelete, parentFiber, error);
-      }
```

Comment:
This catch was removed since it should be handled by the individual calls deeper.

Reply:
Not all host config methods are wrapped because that would be a lot of them so doesn't seem worth it. E.g. the Float ones could error and take other things along with it.